fhir-persistence/src/main/java/com/ibm/fhir/persistence/FHIRPersistence.java

```diff
@@ -293,12 +312,14 @@ default ResourceEraseRecord erase(EraseDTO eraseDto) throws FHIRPersistenceExcep
      * {@link Future} can be used to obtain the status of the operation. If the result
      * is null, then the implementation does not support offloading and the payload must
```

Comment:
I wonder if its worth having `isOffloadingSupported()` similar to how we have `isReindexSupported()`,  `isChangesSupported()` and `isDeleteSupported()`

Reply:
Offloading can be datasource-specific

Reply:
but each datasource gets its own FHIRPersistence.java instance, right?  so maybe the impl of an `isOffloadingSupported()` could check the config?

Reply:
added isOffloadingSupported().